apps/rush-lib/src/cli/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -193,6 +198,13 @@ export class PnpmShrinkwrapFile extends BaseShrinkwrapFile {
     return this._normalizeDependencyVersion(dependencyName, packageDescription.dependencies[dependencyName]);
   }
 
+  protected checkValidVersionRange(dependencyVersion: string, versionRange: string): boolean { // override
+    // dependencyVersion could be of two formats:
+    // 1. (d*).(d*).(d*)
+    // 2. <path>(d*).(d*).(d*) for example /foo/1.0.0
```

Comment:
I forgot to update this comment, will do it in it. IV

Reply:
it. IV

---
In reply to: [183136844](https://github.com/Microsoft/web-build-tools/pull/615#discussion_r183136844) [](ancestors = 183136844)